### PR TITLE
Add more variety to vendor thank you and advertisement messages

### LINKED ISSUE #23803: Write more thank you messages for vending machines
## Description

I implemented custom thank you messages for the different vending machines, but I'm not really a writer and we could use some more messages to add some variety. If anyone wants to do some writing, here's an opportunity to expand on the flavor of each vendor. Feel free to edit or remove the ones I wrote!

**Additional context**

The messages are localized, with the text in Fluent files in Resources/Locale/en-US/advertisements/vending/. The id strings then need to be added to the matching YAML files in Resources/Prototypes/Catalog/VendingMachines/Advertisements/.

### PR DESCRIPTION
**Pull request by UbaserB**
## About the PR

Partially fixes #23803 by adding more thank you messages, however i have not added any new thank you messages to vendors that don't already have unique messages.

### CODE REVIEW

**Review by FairlySadPanda — COMMENTED** [review]
Did a quick SPAG pass (spelling/punctuation/grammar) and caught a few things. Avoided rewording anything except in two instances where the sentence was a bit unclear

**Comment by UbaserB** [comment]
> FYI, you worded the comment of PR using "[...] resolves #issuenumber" and github doesn't care that there was a "partially" preceding that. It will automatically mark that issue as resolved upon merging this PR.
> 
> Not sure if that was desired or not, since "partially resolves ≠ resolves". Just pointing it out.

im gonna rename resolves then

**Comment by UbaserB** [comment]
Fixed.

File: Resources/Locale/en-US/advertisements/vending/coffee.ftl [hunk 1]
```diff
@@ -11,6 +11,8 @@ advertisement-coffee-10 = Coffee helps you work!
 advertisement-coffee-11 = Try some tea.
 advertisement-coffee-12 = We hope you like the best!
 advertisement-coffee-13 = Try our new chocolate!
+advertisement-coffee-14 = Hot beverages! come get yours now!
```
Comment (Partmedia) [inline Resources/Locale/en-US/advertisements/vending/coffee.ftl:14]: Capitalization

File: Resources/Locale/en-US/advertisements/vending/games.ftl [hunk 2]
```diff
@@ -5,7 +5,10 @@ advertisement-goodcleanfun-4 = Roll for initiative!
 advertisement-goodcleanfun-5 = Elves and dwarves!
 advertisement-goodcleanfun-6 = Paranoid computers!
 advertisement-goodcleanfun-7 = Totally not satanic!
-advertisement-goodcleanfun-8 = Fun times forever!
+advertisement-goodcleanfun-8 = Crypts and Carps!
+advertisement-goodcleanfun-9 = Fun times forever!
```
Comment (Partmedia) [inline Resources/Locale/en-US/advertisements/vending/games.ftl:9]: You can swap 8 and 9 to reduce the diff and make it easier for translators.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/ammo.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/ammo.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/boozeomat.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/boozeomat.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/cigs.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/cigs.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/clothesmate.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/clothesmate.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/cola.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/cola.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/curadrobe.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/curadrobe.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/discount.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/discount.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/games.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/games.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/happyhonk.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/happyhonk.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/magivend.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/magivend.yml:1]: It is not necessary to re-indent everything.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/smartfridge.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/smartfridge.yml:1]: It is not necessary to re-indent everything.

**Review by Partmedia — CHANGES_REQUESTED** [review]
Thanks for your contribution. Please take a look at the review comments and avoid changing so much indentation.

**Comment by UbaserB** [comment]
Review addressed.

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/condiments.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/condiments.yml:1]: Unnecessary re-indentation

File: Resources/Prototypes/Catalog/VendingMachines/Advertisements/curadrobe.yml
Comment (Partmedia) [inline Resources/Prototypes/Catalog/VendingMachines/Advertisements/curadrobe.yml:1]: Unnecessary re-indentation

**Review by Partmedia — CHANGES_REQUESTED** [review]
There's still some unnecessary re-indentation. You can see these yourself by inspecting the diff.

**Comment by UbaserB** [comment]
> There's still some unnecessary re-indentation. You can see these yourself by inspecting the diff.

ohhhh yeah i kind of forgot lol let me fix them all up

**Comment by UbaserB** [comment]
I made sure to triple check it this time so it should be all good

**Review by Partmedia — APPROVED** [review]

**Comment by Partmedia** [comment]
Thank you for your contribution!

**Pull request merged**